src/Security/Member.php

```diff
@@ -1123,6 +1123,16 @@ public static function set_title_columns($columns, $sep = ' ')
 
     //------------------- HELPER METHODS -----------------------------------//
 
+    /**
+     * Simple prxoy method to get the Surname property of the member
```

Comment:
@Hels666 pointed out my typo...

```suggestion
     * Simple proxy method to get the Surname property of the member
```